llvm/lib/Transforms/InstCombine/InstCombineCompares.cpp

```diff
@@ -7867,6 +7867,51 @@ static Instruction *foldFCmpFNegCommonOp(FCmpInst &I) {
   return new FCmpInst(Pred, Op0, Zero, "", &I);
 }
 
+static Instruction *foldFCmpFSubIntoFCmp(FCmpInst &I, Instruction *LHSI,
+                                         Constant *RHSC, InstCombinerImpl &CI) {
+  const CmpInst::Predicate Pred = I.getPredicate();
+  Value *X, *Y;
```

Comment:
Nit: could just initialize X and Y to LHSI->getOperand() here and use them below. The call to match(LHSI, ...) below is mostly pointless since we already know that LHSI is an fsub.

Reply:
done.